mon-pix/app/styles/pages/_skill-review.scss

```diff
@@ -22,15 +22,32 @@
     align-items: center;
   }
 
-  &__result-container {
+  &__result-detail-container,
+  &__result-abstract-container,
+  &__organization-message {
     padding: 40px 10px;
-    margin-top: -160px;
+    box-shadow: 0 7px 14px 0 rgba(12, 22, 58, 0.1), 0 3px 6px 0 rgba(0, 0, 0, 0.1);
 
     @include device-is('tablet') {
       padding: 32px 48px;
     }
   }
 
+  &__result-abstract-container {
+    margin-top: -160px;
+  }
+
+  &__organization-message {
+    border-radius: 32px 0 32px 32px;
+    box-shadow: 0 7px 14px 0 rgba(12, 22, 58, 0.1), 0 3px 6px 0 rgba(0, 0, 0, 0.1);
+    display: flex;
+    padding: 32px;
+
+    @include device-is('mobile') {
+      flex-direction: column;
+    }
```

Comment:
**suggestion** : c'est une bonne pratique de fonctionner en mobile-first, du coup ce qu'on modifierait dans un `@include device-is` ce serait plutôt pour tablette et desktop.
ça donnerait : 
```
  &__organization-message {
    display: flex;
    flex-direction: column;
    ...

    @include device-is('tablet') {
      flex-direction: row;
    }
```